apps/base/traefik-hub/repository.yaml

```
@@ -5,4 +5,4 @@ metadata:
   namespace: traefik-hub
 spec:
   interval: 5m
-  url: https://traefik.github.io/charts
+  url: https://helm.traefik.io/traefik
```

```suggestion
  url: https://traefik.github.io/charts
```
This is the correct address, as documented in [README of the chart](https://github.com/traefik/traefik-helm-chart?tab=readme-ov-file#prerequisites).